doxygen warning

```
Opening '/Users/runner/work/OpenMS/OpenMS/OpenMS/bld/doc/doxygen/doxygen-error.log' to check for doxygen errors...
----------- ERRORS/WARNINGS -----------------
/Users/runner/work/OpenMS/OpenMS/OpenMS/src/openms/include/OpenMS/METADATA/ExperimentalDesign.h:189: warning: expected <td>, <th> or <tr> tag but found <br> instead!
---------------------------------------------
Skipped over 24 lines with unavoidable warnings
```

--- a/src/openms/include/OpenMS/METADATA/ExperimentalDesign.h
+++ b/src/openms/include/OpenMS/METADATA/ExperimentalDesign.h
@@ -101,11 +101,11 @@ namespace OpenMS
     </tr>
     <tr>
         <td>...</td>
-        <td>...<br></td>
         <td>...</td>
-        <td>...<br></td>
-        <td>...<br></td>
-        <td>...<br></td>
+        <td>...</td>
+        <td>...</td>
+        <td>...</td>
+        <td>...</td>
     </tr>
     <tr>
         <td>22</td>
@@ -168,10 +168,10 @@ namespace OpenMS
     </tr>
     <tr>
         <td>...</td>
-        <td>...<br></td>
         <td>...</td>
-        <td>...<br></td>
-        <td>...<br></td>
+        <td>...</td>
+        <td>...</td>
+        <td>...</td>
     </tr>
     <tr>
         <td>22</td>
@@ -200,8 +200,8 @@ namespace OpenMS
     </tr>
     <tr>
         <td>...</td>
-        <td>...<br></td>
-        <td>...<br></td>
+        <td>...</td>
+        <td>...</td>
     </tr>
     <tr>
         <td>22</td>

